tests_e2e/azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+variables:
+  - name: azureConnection
+    value: 'AzLinux DCR Public (8e037ad4-618f-4466-8bc8-5099d41ac15b)'
+  - name: subId
+    value: '8e037ad4-618f-4466-8bc8-5099d41ac15b'
+  - name: testsSourcesDirectory
+    value: "$(Build.SourcesDirectory)/tests_e2e"
+
+trigger:
+  - develop
+
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
```

Comment:
is this pool for orchestrator vms?

Reply:
Yes, that is the pool of Azure Pipelines VMs where we run the orchestrator (singular, now we have only 1 orchestrator)

Reply:
you mean you plan to keep only 1 for all scenario or at this point we have one?

Reply:
1 at this point. the actual number will depend on how we organize the test runs and how much we can take advantage of LISA's capabilities.